### PR TITLE
RD-2934 Add "Service button" widget documentation page

### PR DESCRIPTION
**Pull request by qooban**
Documentation for new widget developed in https://github.com/cloudify-cosmo/cloudify-stage/pull/1555.

### CODE REVIEW

File: content/working_with/console/widgets/serviceButton.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Settings
+
+* `Color` - The color of the button. Available icons list can be found
+  at: [Theming - Semantic UI React](https://react.semantic-ui.com/layouts/theming). Default: ''
+* `Label` - The label displayed in the button. Default: 'Create a service'
+* `Icon` - Name of the icon displayed in the button. Available icons list can be found
+  at: [Icon - Semantic UI React](https://react.semantic-ui.com/elements/icon). Default: 'add'
+* `Marketplace tabs` - List of tab name and marketplace URL pairs. For the expected format of the data provided in 
+  the URL, see `Blueprint Examples URL` configuration field in
```
Comment (kubama) [inline content/working_with/console/widgets/serviceButton.md:24]: I think 'the expected format of the data provided in the URL' is misleading, IIUC it refers to the structure of the JSON file located at the URL, not the data that is part of the URL itself.

File: content/working_with/console/widgets/serviceButton.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+layout: bt_wiki
+title: Service button
+category: Widgets
+draft: false
+---
+
+Displays a button allowing to upload a blueprint from the configured blueprint catalog/marketplace.
```
Comment (kubama) [inline content/working_with/console/widgets/serviceButton.md:8]: Looks like we are using terms 'catalog' and 'marketplace' interchangeably. Would be good to decide to use only one of them consistently.

Comment (qooban) [reply content/working_with/console/widgets/serviceButton.md:8]: I know, it's hard to say which one would be the bets here. Blueprint Catalog widget and changes in Blueprints widget should be documented first and then I could refer to it probably.

I'll go with 'marketplace' for now.

File: content/working_with/console/widgets/serviceButton.md (hunk 1 above)
Comment (qooban) [reply content/working_with/console/widgets/serviceButton.md:24]: I'll remove 'the expected format' part right now. I think I'll need to wait for Blueprint Catalog and Blueprints widget update first.

**Review by alexmolev — APPROVED** [review]

**Review by kubama — APPROVED** [review]

**Pull request merged**